cmd/up/up.go

```diff
@@ -467,10 +471,16 @@ func (up *upContext) getCurrentDeployment(ctx context.Context, autoDeploy bool)
 		return nil, false, err
 	}
 
-	if !autoDeploy {
-		if err := utils.AskIfDeploy(up.Dev.Name, up.Dev.Namespace); err != nil {
-			return nil, false, err
+	if !up.Dev.Autocreate {
+		err = errors.UserError{
+			E: fmt.Errorf("Deployment %s not found in namespace '%s'", up.Dev.Name, up.Dev.Namespace),
+			Hint: `Please:
+	- Make sure your application is deployed and ready.
+	- Make sure your context is pointing to the right namespace.
+	- Set the 'autocreate' property in your okteto.yml manifest if you want to automatically create a new development container.
+    More information is available here: https://okteto.com/docs/reference/cli#up`,
```

Comment:
I think this link should point to the `autocreate` field docs, and add it to the third bullet point.

Reply:
@pchico83 I talked to @jLopezbarb about this. I'd like to start adding links to the full documentation of the commands in case of error. I suggested this so users can read about the general workings of the UP command (including a link to the `autocreate` but from the docs themselves), instead of having many links on the same output.

We should then also add a paragraph explaining the 'autocreate' in general in the UP description and add a link to how to configure it in the manifest from there.